util/validateparam.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"fmt"
+	"log"
+
+	"strings"
+)
+
+func ValidateStringParam(paramName string, paramValue string, validValues []string) {
+	valueFound := false
+
+	for i := 0; i < len(validValues); i++ {
```

Comment:
`range` is advisable over indexed loops in golang. 
also, you can optimize the loop a bit with max-two assignments regardless of array size:
```
valueFound := false
for _, validValueItem := range validValues {
  if paramValue == validValueItem {
    valueFound = true
    break
  }
}
```
I didn't test this though but it should work.

Sidenote: Not a blocker but usually using hashes/maps is much faster for checking "if item in expected list" than arrays and iteration.

Reply:
Looks good! Also changed to use hashmap. I was thinking the syntax was better w/ an array but it's really not since you have to do a whole loop to iterate.